pages/input.html

```diff
@@ -0,0 +1,120 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<link rel="stylesheet" href="../css/index.css" />
+		<script
+			defer
+			type="module"
+			src="../javascript/index.js"
+			crossorigin="anonymous"
+		></script>
+		<title>Ueye</title>
+	</head>
+	<body>
+		<div class="root">
+			<nav class="nav">
+				<a class="nav-logo" href="../index.html">
+					<img
+						src="../images/ueye_logo.png"
+						alt="Ueye Logo"
+						class="nav-logo-image"
+					/>
+					<!-- <a>
+						<h1 class="nav-title">Ueye</h1>
+					</a> -->
```

Comment:
Please remove unnecessary code.

Reply:
I am yet to decide if I have to go with just an image as logo or use image + name as logo, will remove it once I decide.

Reply:
Got it.